patches/slonik@30.1.2.patch

```diff
@@ -1,30 +0,0 @@
-diff --git a/dist/src/routines/executeQuery.js b/dist/src/routines/executeQuery.js
```

Comment:
@kamilkisiela I think it's fine?

Reply:
`get-stack-trace` was causing some damage but I don't remember why. I add comments now to remember but this patch was done a year ago or something.

Reply:
I think we should keep using this patch, just in case and once we figure out if we should use Slonik (upgrade to the latest) or migrate to some other library, we will decide if we want to keep it or not.

Reply:
Yeah I think it seems fine now, I also remember an issue we had but I wasn't sure how to reproduce?

Reply:
"it seems fine now" maybe :) We'll see when we deploy it to production